diff --git a/node_modules/@project-serum/anchor/dist/esm/program/common.js b/node_modules/@project-serum/anchor/dist/esm/program/common.js
index 164d8ff..bab8614 100644
--- a/node_modules/@project-serum/anchor/dist/esm/program/common.js
+++ b/node_modules/@project-serum/anchor/dist/esm/program/common.js
@@ -38,15 +38,6 @@ export function validateAccounts(ixAccounts, accounts = {}) {
 }
 // Translates an address to a Pubkey.
 export function translateAddress(address) {
-    if (typeof address === "string") {
-        const pk = new PublicKey(address);
-        return pk;
-    }
-    else if (address.constructor.prototype.constructor.name === "PublicKey") {
-        return address;
-    }
-    else {
-        throw new Error("Given address is not a PublicKey nor a string.");
-    }
+    return new PublicKey(address);
 }
 //# sourceMappingURL=common.js.map
\ No newline at end of file
